docs/spark/tutorials/hdinsight-deployment.md

```diff
@@ -9,124 +9,180 @@ ms.custom: mvc
 
 # Deploy a .NET for Apache Spark application to Azure HDInsight
 
-This tutorial teaches how to deploy a .NET for Apache Spark application to Azure HDInsight.
+This tutorial teaches how to deploy your .NET for Apache Spark app to the cloud through an Azure HDInsight cluster. HDInsight makes it easier to create and configure a Spark cluster in Azure since Spark clusters in HDInsight are compatible with Azure Storage and Azure Data Lake Storage. 
 
 In this tutorial, you learn how to:
 
 > [!div class="checklist"]
 >
-> * Prepare Microsoft.Spark.Worker
-> * Publish your Spark .NET app
-> * Deploy your app to Azure HDInsight
-> * Run your app
+> * 
```

Comment:
Is this list deleted by mistake ?
If not, then the `In this tutorial, you lean how to:` should be deleted too.